backend/src/schema/resolvers/users.js

```diff
@@ -30,6 +30,7 @@ export const getBlockedByUsers = async context => {
     .relationship(userModel.relationships().get('blocked'))
     .to('blocked', userModel)
     .where('blocked.id', context.user.id)
+    .where('blocked.role', 'user')
```

Comment:
Ah, **that's** the line! Now I get it.

Reply:
@mattwr18 I would prefer you could add a guard clause and return empty array if `context.user.role > 'user'` or something like that. It took me a while to understand that the additional `where('blocked.role')` is excluding *your own user node* from the result set, thus returning empty array. We could make this more explicit by returning empty array for this case.

Reply:
A guard clause would also save us one unnecessary database call.